p4sdk/images/es2k_skip_p4.conf.template

```diff
@@ -0,0 +1,41 @@
+{
+    "chip_list": [
+    {
+        "id": "asic-0",
+        "chip_family": "mev",
+        "instance": 0,
+        "pcie_bdf": "$CPF_BDF",
+        "iommu_grp_num": $IOMMU_GROUP
+    }
+    ],
+    "instance": 0,
+    "cfgqs-idx": "0-15",
+    "p4_devices": [
+    {
+        "device-id": 0,
+        "fixed_functions" : [],
+        "eal-args": "--lcores=1-2 -a $CPF_BDF,vport=[0-1] -- -i --rxq=1 --txq=1 --hairpinq=1 --hairpin-mode=0x0",
+        "p4_programs": [
+        {
+            "program-name": "linux_networking",
+            "bfrt-config": "/root/linux_networking/linux_networking.bf-rt.json",
+            "p4_pipelines": [
+            {
+                "p4_pipeline_name": "main",
+                "context": "/root/linux_networking/linux_networking.context.json",
+                "config": "/root/linux_networking/tofino.bin",
```

Comment:
This file is missing from the p4 artifacts - this file is an empty file and should be created before starting infrap4d

Reply:
> This file is missing from the p4 artifacts - this file is an empty file and should be created before starting infrap4d

I will add "create" this file.